cmds/loopqueue.js

```diff
@@ -0,0 +1,71 @@
+const { MessageButton, MessageActionRow } = require("gcommands");
+const { MessageEmbed } = require("discord.js");
+
+module.exports = {
+    name: "loop",
```

Comment:
It would be better if you put it in one loop command and put modes like /loop mode:queue/song .

Reply:
ok

Reply:
I don't have it tested, but I hope, this will work.

Reply:
wait much commented on file

Reply:
ok good

Reply:
and now the replies...

Reply:
now good i hope